price-feeder/config/config.go

```diff
@@ -117,32 +117,32 @@ type (
 	// Telemetry defines the configuration options for application telemetry.
 	Telemetry struct {
```

Comment:
This config struct is mostly just necessary for processing the toml files. Added the mapstructure tags so that we can just decode directly into the sdk config struct.